pkgs/top-level/all-packages.nix

```diff
@@ -16495,7 +16495,9 @@ in
       virtualbox = pkgs.virtualboxHardened;
     };
 
-    wireguard = callPackage ../os-specific/linux/wireguard { };
+    wireguard = if (
+      (stdenv.lib.versionAtLeast kernel.version "3.10") && (stdenv.lib.versionOlder kernel.version "5.6")
```

Comment:
```suggestion
      (lib.versionAtLeast kernel.version "3.10") && (lib.versionOlder kernel.version "5.6")
```

Reply:
stdenv contains lib for convenience, but since you're at the top-level package scope, it's available anyway.

Reply:
done. thx :slightly_smiling_face: